pylint/__pkginfo__.py

```diff
@@ -40,7 +40,7 @@
     version += "-dev" + str(dev_version)
 
 install_requires = [
-    "astroid>=2.4.0,<=2.5",
+    "astroid>=2.5.0,<=2.6",
```

Comment:
I think we need to keep the version as is for now (or increase the upper bound limit to 2.6) since we have not released the latest `astroid` version. This leads to spurious failures on the CI side.

Reply:
:+1: